homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -83,7 +87,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     eagle_data = EagleData(eagle_reader)
     eagle_data.update()
-    monitored_conditions = list(SENSORS)
+    monitored_conditions = config[CONF_MONITORED_CONDITIONS]
```

Comment:
Please move this up to the other local variables that store config values e.g. line 81.